mllib/src/main/scala/org/apache/spark/mllib/clustering/GaussianMixture.scala

```diff
@@ -43,6 +43,9 @@ import org.apache.spark.util.Utils
  * While this process is generally guaranteed to converge, it is not guaranteed
  * to find a global optimum.
  *
+ * Computation of the Gaussian pdfs are distributed across workers when dimensionality of the input
+ * data >= 30 and number of centers >= 10.
```

Comment:
This needs update. I don't think we need to mention the explicit rule we use in public doc. This is implementation detail.